### PR TITLE
[Docs] Add tutorials category to integrations and reorganize integrations categories and sidebar

### PR DESCRIPTION
**Pull request by neverett**
## Summary & Motivation

* Add "Tutorials" category to "Integrations"
* Reorganize categories for integrations libraries -- we now have "Dagster supported" and "Community supported".

## How I Tested These Changes

Tested locally

## Changelog

> Insert changelog entry or delete this section.

### CODE REVIEW

**Comment by cmpadden** [comment]
@neverett looks like we're getting a couple errors in Vercel:
```
        Error: Can't render static file for pathname "/integrations/"
            at generateStaticFile (/vercel/path0/docs/docs-beta/node_modules/@docusaurus/core/lib/ssg.js:119:15)
            at runNextTicks (node:internal/process/task_queues:65:5)
            at process.processImmediate (node:internal/timers:459:9)
            at async /vercel/path0/docs/docs-beta/node_modules/p-map/index.js:57:22 {
          [cause]: Error: Unexpected: cant find current sidebar in context
              at useCurrentSidebarCategory (server.bundle.js:123253:238)
              at DocCardListForCurrentSidebarCategory (server.bundle.js:123502:125)
              at Uc (server.bundle.js:130680:44)
              at Xc (server.bundle.js:130682:253)
              at Z (server.bundle.js:130688:89)
              at Xc (server.bundle.js:130682:481)
              at Z (server.bundle.js:130688:89)
              at Xc (server.bundle.js:130682:481)
              at Z (server.bundle.js:130688:89)
              at Xc (server.bundle.js:130686:231)
        },
        Error: Can't render static file for pathname "/integrations/aws/"
            at generateStaticFile (/vercel/path0/docs/docs-beta/node_modules/@docusaurus/core/lib/ssg.js:119:15)
            at runNextTicks (node:internal/process/task_queues:65:5)
            at process.processImmediate (node:internal/timers:459:9)
            at async /vercel/path0/docs/docs-beta/node_modules/p-map/index.js:57:22 {
          [cause]: Error: Unexpected: cant find current sidebar in context
              at useCurrentSidebarCategory (server.bundle.js:123253:238)
              at DocCardListForCurrentSidebarCategory (server.bundle.js:123502:125)
              at Uc (server.bundle.js:130680:44)
              at Xc (server.bundle.js:130682:253)
              at Z (server.bundle.js:130688:89)
              at Xc (server.bundle.js:130682:481)
              at Z (server.bundle.js:130688:89)
              at Xc (server.bundle.js:130682:481)
              at Z (server.bundle.js:130688:89)
              at Xc (server.bundle.js:130686:231)
        },
```

**Comment by neverett** [comment]
@cmpadden those errors should be fixed now, mind taking another look?

File: docs/docs-beta/sidebars.ts [hunk 1]
```diff
@@ -86,135 +86,47 @@ const sidebars: SidebarsConfig = {
...
-        },
-        {
-          type: 'category',
-          label: 'Monitoring',
-          items: ['integrations/prometheus', 'integrations/datadog', 'integrations/aws/cloudwatch'],
-        },
-        {
-          type: 'category',
-          label: 'Alerting',
+          label: 'Dagster-supported',
```
Comment (cmpadden) [inline docs/docs-beta/sidebars.ts:106]: What do you think about making `Dagster-supported` the default, so it's at the top level, and then community supported being a sub folder? I think that might reduce the visual clutter slightly.

Comment (cmpadden) [reply docs/docs-beta/sidebars.ts:106]: And then we can create some cool React components to have tags on the integration pages themselves with `dagster verified` or something along those lines.

Comment (neverett) [reply docs/docs-beta/sidebars.ts]: I think that's a great idea! Looks like Docusaurus has tagging built in, so I just used that. All integrations libraries now live in the "Libraries" section and have relevant tags (dagster-supported, community-supported, ETL, etc); tags are at the bottom of the page.

**Review by cmpadden — APPROVED** [review]
I was able to confirm that things look good running locally.

There are a couple of broken links, but approving so that you're unblocked.

**Comment by neverett** [comment]
> I was able to confirm that things look good running locally.
> 
> There are a couple of broken links, but approving so that you're unblocked.

Ugh, sorry, I should've waited for the build to finish before commenting -- I'll fix those. 

Also cc @dehume -- here's more docs revamp work, if you're curious

**Pull request merged**